src/test/java/seedu/bookmark/logic/commands/EditCommandTest.java

```diff
@@ -54,15 +57,15 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
         Index indexLastPerson = Index.fromOneBased(model.getFilteredBookList().size());
```

Comment:
Some "person" variables can still be changed to "book"

Reply:
Thank you for pointing it out!